Hybrid-state.md

```diff
@@ -0,0 +1,82 @@
+
+2. **If contributor wants to modify a suite** 
+
+  * If the suite is migrated to webdriverIO then make the changes in webdriverIO.
+  
+  * If the suite is still in protractor then make the changes in protractor.
+
+    **Edge Case**: If we already have a migration PR open for a particular suite and at the same time any other contributor also made a PR adding the tests in the same suite then, if migration's PR will be merged first, the contributor needs to write the tests in webdriverIO (contributor can take help from [Shivam Jha](#contact)).  Otherwise, migration PR needs to be updated with the changes.
+
+    Also, if migration PR will get merged first then everyone will be notified about it accordingly as migration PR will have the **PR: that requires post-merge sync to the HEAD label.**
+
+3. **If contributor is planning to make some changes in the common files**
+
+   The tests might break if the changes are only applied in one version of the common file. So we need to make sure to keep the two versions of the common file in sync. We will not merge the PR of the contributor if it's not synced. Contributor can take help from [Shivam Jha](#contact) or the migration guide in order to make the necessary changes for the other version.
+  
+#### Current approach to migrate the dependencies
+
+To reduce the duplicate code in the codebase we are going to migrate only those portions of the dependencies which are used in the suite that we are migrating. The remaining portions will be migrated when a suite which depends upon them is being migrated.
+
+Let's understand this situation with a simple test case.
+
+We have two test files:
+
+|**File**            | **Dependencies**         |
+|--------------------|--------------------------|
+|AdditonalPlayer     |ExplorationEditorPage, ExplorationPlayerPage, LibraryPage|
+|AdditonalEditorFeaturesModals| ExplorationEditorPage |
+
+**Common Files** : ExplorationEditorPage
+
+* While migrating **AdditionalPlayer** we will migrate its dependencies, there will be no issue with **ExplorationPlayerPage**, **LibraryPage** as they do not have any other suites dependent on them, we will just migrate them and delete their protractor version. But for **ExplorationEditorPage** we still have one dependent suite i.e. **AdditonalEditorFeatures**, so we need to migrate it to successfully migrate **AdditionalPlayer** suite, but we cannot delete the protractor version (as a protractor suite cannot work on webdriverIO util files).
+
+* So now the **ExplorationEditorPage** file will be a common file that presents in both webdriverIO and protractor version. To reduce code duplicity to some extent I will migrate only that portion of ExplorationEditorPage that will be used in the **AdditonalPlayer** suite, rest can be migrated when we will migrate **AdditonalEditorFeaturesModals**.
```

Comment:
duplicity -> duplication

"duplicity" means "deceitfulness"

Also, you switch to "I" here instead of the "we" you've been using throughout

Reply:
Updated.